lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -23,7 +23,7 @@ function multiplyFractions(node) {
     return Node.Status.noChange(node);
   }
   const atLeastOneFraction = node.args.some(
-    arg => Node.Type.isOperator(arg, '/'));
+    arg => Node.Type.isFraction(arg));
```

Comment:
oh I now know how to make this cleaner (if only past me knew!)

`node.args.some(Node.Type.isFraction)`